addons/survey/models/survey_survey.py

```diff
@@ -434,6 +495,70 @@ def _get_survey_questions(self, answer=None, page_id=None, question_id=None):
             questions = questions & answer.predefined_question_ids
         return questions, page_or_question_id
 
+    # ------------------------------------------------------------
+    # SESSIONS MANAGEMENT
+    # ------------------------------------------------------------
+
+    def session_next_question(self):
+        """ Triggers the next question of the session.
+
+        We artificially add 2 seconds to the 'current_question_start_time' to account for server delay.
+        As the timing can influence the attendees score, we try to be fair with everyone by giving them
+        an extra few seconds before we start counting down.
+
+        Frontend should take the delay into account by displaying the appropriate animations. """
+
+        self.ensure_one()
+
+        if not self.question_ids:
+            return
+
+        if not self.session_current_question_id:
+            question = self.question_ids[0]
+        else:
+            questions = list(enumerate(self.question_ids))
+            current_question_index = questions.index(
```

Comment:
Unfortunately, `next_page_or_question` is based on a `user_input` that progresses through the survey.

Here I have no `user_input`, just the host going to the next question of the "session" (which is not a model anymore).

Making `next_page_or_question` have an optional user_input could be an idea but I don't know if it's worth it? Maybe it will just move the complexity.